src/main/webapp/js/finscholar/scholarshippageview/scholarshippageview.js

```diff
@@ -41,23 +41,20 @@ class ScholarshipPageView {
    * Render the scholarship page.
    * @public
    * @param {!Element} container - The DOM element for single scholarship page.
+   * @param {string} id - The unique id of the scholarship.
    */
-  async renderScholarship(container) {
-    // In the prototype, the id is set to 0 by default. Later we'll pass in id as parameter.
-    const id = 0;
+  async renderView(container, id) {
```

Comment:
It seems right now it's just rendering the view for scholarships, will this be generalized for the other views?

Reply:
I'll make a common parent class for scholarship page view and college page view.